docs/docs/behaviors/macros.md

```diff
@@ -0,0 +1,168 @@
+				, <&macro_tap>
+				, <&kp TAB>
+                , <&macro_pause_for_release>
+				, <&macro_release &kp LSHFT>
+				;
+		};
+    };
+};
+```
+
+The macro can then be bound in your keymap by referencing it by the label `&alt_tab`.
+
+### Bindings
+
+Like hold-taps, macros are created by composing other behaviors, and any of those behaviors can
+be added to the `bindings` list.
+
+### Wait Time
+
+The wait time setting controls how long of a delay is introduced between activations of each behavior in the macro bindings sequence.
+The default wait time is 100ms, but the value can be overridden by setting the `wait-ms` property on the macro, e.g.:
+
+```
+/ {
+    behaviors {
+        ab_macro: ab_macro {
+			label = "AB_MAC";
+			compatible = "zmk,behavior-macro";
+			#binding-cells = <0>;
+            wait-ms = <30>;
+			bindings
+				= <&kp A &kp B>
+				;
+		};
+    };
+};
+```
+
+### Tap Time
+
+The tap time setting controls how long of a delay is introduced between press and release of a tapped behavior in the macro bindings sequence.
+The default tap time is 100ms, but the value can be overridden by setting the `tap-ms` property on the macro, e.g.:
+
+```
+/ {
+    behaviors {
+        ab_macro: ab_macro {
+			label = "AB_MAC";
+			compatible = "zmk,behavior-macro";
+			#binding-cells = <0>;
+            tap-ms = <30>;
+			bindings
+				= <&kp A &kp B>
+				;
+		};
+    };
+};
+```
+
+## Macro Controls
+
+There are a set of special macro controls that can be included in the `bindings` list to modify the
+way the macro sequence is processed.
+
+### Binding Activation Mode
+
+Bindings in a macro sequence are activated differently, depending on the current "activation mode" of the macro.
+
+Available modes:
+
+- Tap - The default mode; when in this mode, the macro processing will press, then release, each behavior the macro sequence. This mode is useful for
+  basic keycode output to hosts, i.e. when activating a `&kp` behavior.
+- Press - In this mode, the macro processing will only trigger a press on the behaviors in the macro sequence. This is useful for holding down modifiers for some duration of a macro, e.g. `&kp LALT`.
+- Release - In this mode, the macro processing will only trigger a release on the behaviors in the macro sequence. This is useful for releasing modifiers previously pressed earlier in the macro sequence processing, e.g. `&kp LALT`.
+
+To modify the activation mode, macro controls can be added at any point in the bindings sequence.
+
+- `&macro_tap`
+- `&macro_press`
+- `&macro_release`
+
+A concreate example, used to hold a modifier, tap multiple keys, then release the modifier, would look like:
+
+```
+bindings
+    = <&macro_press &kp LSHFT>
+    , <&macro_tap &kp Z &kp M &kp K>
+    , <&macro_release &kp LSHFT>
+    ;
+```
+
+### Processing Continuation on Release
+
+The macro sequence can be paused, so that only part of the sequence is processed when the macro is pressed, and the remainder is processed once
```

Comment:
```suggestion
The macro can be paused so that only part of the `bindings` list is processed when the macro is pressed, and the remainder is processed once
```